### ISSUE TITLE
Detecting hand drawn lines: training causing kernel to crash at epoch 1

### ISSUE THREAD

**Issue by jowparks** [matterport/Mask_RCNN#2068]
I am trying to train a simple model to recognize and segment hand drawn lines. Each image in my repo contains one line, so I generate the mask by just doing a threshold (which seems to work well based on inspection of masks).

https://github.com/jowparks/mrcnn_draw_lines
(ve I am using for testing has requirements.txt installed)

For some reason the model building fails when I start training at first epoch, the kernel just crashes. I tried using a few different version combinations of tensorflow/keras and the `shapes` dataset works fine for both combos, but my training fails by either throwing a StopIteration exception
```
if all_finished and self.queue.empty():
--> 710                     raise StopIteration()
```
 or my python kernel continuously crashes  (mac level error warning saying `python3 unexpectedly quit`)

Any ideas? Thanks!

**Comment by nataliameira** [comment]
@jowparks 

Did you solve this problem?
